site/src/sources/lion.json

```diff
@@ -6,15 +6,36 @@
   "url": "https://github.com/ing-bank/lion",
   "by": "ING",
   "components": [
+    {
+      "name": "Accordion",
+      "url": "https://lion-web.netlify.app/components/accordion/"
+    },
+    {
+      "name": "Button",
+      "url": "https://lion-web.netlify.app/components/button/"
+    },
+    {
+      "name": "Calendar",
+      "url": "https://lion-web.netlify.app/components/calendar/"
+    },
+    {
+      "name": "Checkbox Group",
+      "url": "https://lion-web.netlify.app/components/checkbox-group/"
+    },
+    {
+      "name": "Collapsible",
+      "openUIName": "Collapse",
+      "url": "https://lion-web.netlify.app/components/collapsible/"
+    },
     {
       "name": "Combobox",
       "openUIName": "Select",
-      "url": "https://lion-web.netlify.app/components/inputs/combobox/overview/"
+      "url": "https://lion-web.netlify.app/components/inputs/combobox/"
```

Comment:
This URL seems to be wrong, at least some components notably combobox does seem to require the /overview/ on the end.

Also I think the domain is https://lion.js.org (at least that's what shows on Google?)